tests/test_pipelines_image_segmentation.py

```diff
@@ -0,0 +1,282 @@
+        model_id = "facebook/detr-resnet-50-panoptic"
+
+        image_segmenter = pipeline("image-segmentation", model=model_id)
+
+        outputs = image_segmenter("http://images.cocodataset.org/val2017/000000039769.jpg")
+        outputs["png_string"] = hashlib.sha1(outputs["png_string"].encode("UTF-8")).hexdigest()
+        self.assertEqual(
+            nested_simplify(outputs, decimals=4),
+            {
+                "segments_info": [
+                    {"id": 0, "score": 0.9094, "label": "blanket"},
+                    {"id": 1, "score": 0.9941, "label": "cat"},
+                    {"id": 2, "score": 0.9987, "label": "remote"},
+                    {"id": 3, "score": 0.9995, "label": "remote"},
+                    {"id": 4, "score": 0.9722, "label": "couch"},
+                    {"id": 5, "score": 0.9994, "label": "cat"},
+                ],
+                "png_string": "d0c3d58467818e568604e369c945b18ce05e28c0",
+            },
+        )
+
+        outputs = image_segmenter(
+            [
+                "http://images.cocodataset.org/val2017/000000039769.jpg",
+                "http://images.cocodataset.org/val2017/000000039769.jpg",
+            ],
+            threshold=0.0,
+        )
+        for o in outputs:
+            o["png_string"] = hashlib.sha1(o["png_string"].encode("UTF-8")).hexdigest()
+
+        self.assertEqual(
+            nested_simplify(outputs, decimals=4),
+            [
+                {
+                    "segments_info": [
+                        {"id": 0, "score": 0.9094, "label": "blanket"},
+                        {"id": 1, "score": 0.9941, "label": "cat"},
+                        {"id": 2, "score": 0.9987, "label": "remote"},
+                        {"id": 3, "score": 0.9995, "label": "remote"},
+                        {"id": 4, "score": 0.9722, "label": "couch"},
+                        {"id": 5, "score": 0.9994, "label": "cat"},
+                    ],
+                    "png_string": "d0c3d58467818e568604e369c945b18ce05e28c0",
+                },
+                {
+                    "segments_info": [
+                        {"id": 0, "score": 0.9094, "label": "blanket"},
+                        {"id": 1, "score": 0.9941, "label": "cat"},
+                        {"id": 2, "score": 0.9987, "label": "remote"},
+                        {"id": 3, "score": 0.9995, "label": "remote"},
+                        {"id": 4, "score": 0.9722, "label": "couch"},
+                        {"id": 5, "score": 0.9994, "label": "cat"},
+                    ],
+                    "png_string": "d0c3d58467818e568604e369c945b18ce05e28c0",
+                },
+            ],
+        )
+
+    @require_torch
+    @slow
+    def test_subtask(self):
+        model_id = "facebook/detr-resnet-50-panoptic"
+
+        image_segmenter = pipeline("image-segmentation", model=model_id)
+
+        outputs = image_segmenter("http://images.cocodataset.org/val2017/000000039769.jpg", subtask="panoptic")
```

Comment:
This is EXACTLY the same as no `subtask`. What is the point of having subtasks ?